src/main/java/hudson/plugins/emailext/plugins/content/FailedTestsContent.java

```diff
@@ -64,21 +65,22 @@ public boolean acceptsMacroName(String macroName) {
 
 
     @Override
-    public String evaluate(AbstractBuild<?, ?> build, TaskListener listener, String macroName) {
+    public String evaluate(AbstractBuild<?, ?> build, TaskListener listener, String macroName) throws MacroEvaluationException{
         return evaluate(build, build.getWorkspace(), listener, macroName);
     }
 
     @Override
-    public String evaluate(Run<?, ?> run, FilePath workspace, TaskListener listener, String macroName) {
+    public String evaluate(Run<?, ?> run, FilePath workspace, TaskListener listener, String macroName) throws MacroEvaluationException {
         AbstractTestResultAction<?> testResult = run.getAction(AbstractTestResultAction.class);
         SummarizedTestResult result = prepareSummarizedTestResult(testResult);
 
         switch (outputFormat) {
             case "yaml":
                 try {
                     return result.toYamlString();
-                } catch (Exception e) { }
-                return "Bad format";
+                } catch (Exception e) {

```

Comment:
The only checked exception that can be thrown is `JsonProcessingException` so why catch the generic exception? It adds no benefit in this scenario and is generally a bad practice (anti-pattern).